examples/va/nfv/ovs-dpdk-sriov/service-values.yaml

```diff
@@ -24,6 +24,14 @@ data:
       vhost_sock_dir = /var/lib/vhost_sockets
       [ml2_type_vlan]
       network_vlan_ranges = dpdk1:206:209,dpdk2:206:209,sriov1,sriov2 # CHANGEME
+      [neutron]
+      physnets = dpdk1, dpdk2 # CHANGEME
+      [neutron_physnet_dpdk1] # CHANGEME
+      numa_nodes = 0 # CHANGEME
+      [neutron_physnet_dpdk2 # CHANGEME
+      numa_nodes = 0 # CHANGEME
+      [neutron_tunnel] # CHANGEME
+      numa_nodes = 0 # CHANGEME
```

Comment:
Same question here.  Is this change supposed to be included here?  It's changing a VA, but this PR is adding a new DT.

Reply:
I thought it will not harm to have this. Thanks for pointing. Removing the change